master/buildbot/test/fake/docker.py

```diff
@@ -82,3 +84,7 @@ def create_container(self, image, *args, **kwargs):
 
     def remove_container(self, id, **kwargs):
         del self._containers[id]
+
+
+class APIClient(Client):
```

Comment:
mostly added as a test requirement, I will double check it if it's still needed after refactorization